### ISSUE TITLE
running with VoiceOver turned on: a black rectangle shows up 

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#74537]
while testing #74320 (exploration build) on macOS:

- enable VoiceOver
- activate a text editor 
- press "arrow up" or "arrow down" key

Observe:
a black rectangle shows up at the top of the editor:

![2019-05-29_13-06-49](https://user-images.githubusercontent.com/1898161/58552752-7034aa80-8213-11e9-9d4e-8d07e47b20e0.png)

I've compared this behavior with Insiders and Stable, and the problem already exists there.

**Comment by alexandrudima** [comment]
I think that is some VoiceOver feature. On my machine, I see the rectangle around the entire window:

![image](https://user-images.githubusercontent.com/5047891/62109233-26684f00-b2ac-11e9-8c71-f37c365788a2.png)
